extension/scripts/content/global/ttGlobal.js

```diff
@@ -2,6 +2,7 @@
 
 let highlights;
 let initiatedIconMoving = false;
+let mouseX, mouseY;
```

Comment:
Where is this used? I see it's set but never accessed

Reply:
mouseX is not used, mouseY is used for moving the mini profile.